### PR TITLE
Also warn with more than two dimensions.

### PR DESCRIPTION
**Pull request by RoelvandenBerg**
As discussed. Also warn when more than two dimensions occur.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -45,7 +45,7 @@ The current checks are (see also the 'show-validations' command):
 |       RQ15      | All table geometries must match the geometry_type_name from the gpkg_geometry_columns table. |
 |       RC1       | It is recommended to name all GEOMETRY type columns 'geom'.  |
 |       RC2       | It is recommended to give all GEOMETRY type columns the same name. |
-|       RC3       | It is recommended that multidimensional geometry coordinates (elevation and measurement) contain values. |
+|       RC3       | It is recommended only to use multidimensional geometry coordinates (elevation and measurement) when necessary. |
```
Comment (RoelvandenBerg) [inline README.md:48]:
```suggestion
|       RC3       | It is recommended to only use multidimensional geometry coordinates (elevation and measurement) when necessary. |
```

File: geopackage_validator/validations/geometry_dimension_check.py [hunk 2]
```diff
@@ -30,28 +31,28 @@ def query_dimensions(dataset) -> Iterable[Tuple[str, str]]:
...
                     continue
-                yield table_name, ELEVATION_COORDINATE_NAME
+                yield table_name, ELEVATION_COORDINATE_MESSAGE
 
         dataset.ReleaseResultSet(validations)
 
 
 class GeometryDimensionValidator(validator.Validator):
-    """It is recommended that multidimensional geometry coordinates (elevation and measurement) contain values."""
+    """It is recommended only to use multidimensional geometry coordinates (elevation and measurement) when necessary."""
```
Comment (RoelvandenBerg) [inline geopackage_validator/validations/geometry_dimension_check.py:47]:
```suggestion
    """It is recommended to only use multidimensional geometry coordinates (elevation and measurement) when necessary."""
```

File: tests/validations/test_dimensions_check.py [hunk 3]
```diff
@@ -5,9 +5,14 @@
 def test_with_gpkg():
     # test_dimensions.gpkg has 4 tables,
```
Comment (RoelvandenBerg) [inline tests/validations/test_dimensions_check.py:6]:
```suggestion
```

**Comment by RoelvandenBerg** [comment]
Example output:

```json
{
    "geopackage_validator_version": "0.5.9",
    "start_time": "2021-03-16T13:02:48.745136",
    "duration_seconds": 1,
    "geopackage": "gpkgs/spoorwegen.gpkg",
    "success": true,
    "validations_executed": [
        "RQ1",
        "RQ2",
        "RQ4",
        "RQ5",
        "RQ6",
        "RQ7",
        "RQ9",
        "RQ10",
        "RQ11",
        "RQ12",
        "RQ13",
        "RQ14",
        "RQ15",
        "RQ16",
        "RC1",
        "RC2",
        "RC3"
    ],
    "results": [
        {
            "validation_code": "RC3",
            "validation_description": "It is recommended to only use multidimensional geometry coordinates (elevation and measurement) when necessary.",
            "level": "recommendation",
            "locations": [
                "Table: wissel, has features with more than two dimensions.",
                "Table: spooras, has features with more than two dimensions.",
                "Table: kruising, has features with more than two dimensions.",
                "Table: trace, has features with more than two dimensions."
            ]
        }
    ]
}
```

**Comment by RoelvandenBerg** [comment]
Further implements: https://github.com/PDOK/geopackage-validator/issues/24

**Review by arbakker — APPROVED** [review]

**Pull request merged**